src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -244,12 +219,11 @@ export default class ResultsViewPathwayMapper extends React.Component<
                                 addGenomicDataHandler={
                                     this.addGenomicDataHandler
                                 }
+                                messageBanner={this.renderBanner} //messgaeBanner
                                 tableComponent={this.renderTable}
                                 validGenes={this.validGenes}
-                                toast={toast}
-                            />
-                            <ToastContainer
-                                closeButton={<i className="fa fa-times" />}
+                                toast={null}
```

Comment:
Do we still need to set the toast property? Can't we just remove this line instead?